starknet_py/tests/e2e/client/client_test.py

```diff
@@ -73,7 +73,7 @@ async def test_get_transaction_raises_on_not_received(client):
     with pytest.raises(
         TransactionNotReceivedError, match="Transaction was not received on Starknet."
     ):
-        await client.get_transaction(tx_hash=0x1)
+        await client.get_transaction(tx_hash=0x9999)
```

Comment:
Will it work on say, integration network?

Reply:
Good question. Would a previous test work though?

Reply:
Good question.